ansible/roles/ansible_service_broker_setup/tasks/main.yml

```diff
@@ -72,6 +81,8 @@
         -p REFRESH_INTERVAL={{ broker_bootstrap_refresh_interval }}
         -p TAG={{ apbtag | default('latest') }}
         -p SANDBOX_ROLE={{ broker_sandbox_role }}
+        {% if broker_kind is defined %}-p BROKER_KIND={{ broker_kind }}{% endif %}
```

Comment:
Why not do this for the user if they set coalmine? It's bad enough to have to set 4 entries right now to enable http and I'm thinking either to set those for the user when they run coalmine, or at least cut it down to one ssl option that sets all 4. Having to flip 3 or 4 flags to do one thing is mean. 

```
+  - name: Update Broker Kind and Auth when using coalmine
+    set_fact:
+      cmd_process_asb_template: >-
+        {{ cmd_process_asb_template }}
+        -p BROKER_KIND="ServiceBroker"
+        -p BROKER_AUTH='{ "basic": { "secretRef": { "namespace": "ansible-service-broker", "name": "asb-auth-secret" } } }'
+    when: coalmine is defined and coalmine
```